vignettes/r2dii-analysis.Rmd

```diff
@@ -12,117 +12,121 @@ knitr::opts_chunk$set(
 
 ## Load your r2dii libraries
 
-The first step in your analysis will be to load in the recommended r2dii packages into your current R session. [r2dii.data](https://2degreesinvesting.github.io/r2dii.data) includes fake data to help demonstrate the tool and [r2dii.match](https://2degreesinvesting.github.io/r2dii.match) provides functions 
-to help you easily match your loanbook to asset-level data.
+The first step in your analysis will be to load in the recommended r2dii packages into your current R session. [r2dii.data](https://2degreesinvesting.github.io/r2dii.data) includes fake data to help demonstrate the tool and [r2dii.match](https://2degreesinvesting.github.io/r2dii.match) provides functions to help you easily match your loanbook to asset-level data.
 
-``` {r r2dii libraries}
+``` {r use-r2dii}
 library(r2dii.data)
 library(r2dii.match)
 library(r2dii.analysis)
 ```
 
-We also recommend the following two `tidyverse` packages to manipulate and plot data:
+We also recommend packages in the [tidyverse](https://www.tidyverse.org/); they are optional but useful.
 
-``` {r tidyverse libraries}
-library(dplyr, warn.conflicts = FALSE)
-library(ggplot2)
+``` {r use-tidyverse}
+library(tidyverse)
 ```
 
 ## Match your loanbook to climate-related asset-level data
 
 See [r2dii.match](https://2degreesinvesting.github.io/r2dii.match) for a more complete description of this process. 
 
 ```{r match}
-matched <- match_name(
-  loanbook = loanbook_demo,
-  ald = ald_demo,
-  overwrite = overwrite_demo
-) %>%
-  prioritize()
+# Use these datasets to practice but eventually you should use your own data.
```

Comment:
I think this is more reusable. Users can later change the definitions of `loanbook` and `ald` with their own data and the rest of the code remains untouched.